src/Components/Dashboard.js

```diff
@@ -1,14 +1,14 @@
 import { useState, useEffect } from 'react';
 import React from 'react';
-import ContactCard from './ContactCard';
+import { ContactCard } from './ContactCard';
```

Comment:
Vercel isn't liking this import for some reason, we can look at it for the next push